docs/resources/file.md.erb

```diff
@@ -125,12 +125,30 @@ a user:
 
     it { should be_readable.by_user('user') }
 
+### be_sgid
+
+The `be_sgid` matcher tests if the 'setgid' permission is set on the file or directory.  On executable files, this causes the process to be started owned by the group that owns the file, rather than the primary group of the invocating user.  This can result in escalation of privilege.  On Linux, when setgid is set on directories, setgid causes newly created files and directories to be owned by the group that owns the setgid parent directory; additionally, newly created subdirectories will have the setgid bit set.  To use this matcher:
+
+    it { should be_sgid } 
```

Comment:
Since we kinda want to get rid of the `sgid?` method and go with the clearer `setgid?`, should we advertise the clearer matcher instead?